sklearn/manifold/__init__.py

```diff
@@ -7,6 +7,7 @@
 from .mds import MDS
 from .spectral_embedding_ import SpectralEmbedding, spectral_embedding
 from .t_sne import TSNE
+from .laplacian_eigenmap_ import LaplacianEigenmap, laplacian_eigenmap
 
 __all__ = ['locally_linear_embedding', 'LocallyLinearEmbedding', 'Isomap',
-           'MDS', 'SpectralEmbedding', 'spectral_embedding', "TSNE"]
```

Comment:
Is this to be removed now or when we drop the deprecated class in two releases? Somebody's code will break trying to import the deprecated class now, correct?